staging/src/k8s.io/cli-runtime/pkg/resource/builder_test.go

```diff
@@ -1050,7 +1051,7 @@ func TestRestMappingErrors(t *testing.T) {
 	// ensure that requesting a resource we _know_ not to exist results in an expected *meta.NoKindMatchError
 	err := b.Do().IntoSingleItemImplied(&singleItemImplied).Visit(test.Handle)
 	if err != nil {
-		if !strings.Contains(err.Error(), "server doesn't have a resource type \"foo\"") {
```

Comment:
I think it is better to also have this error check to ensure that there is no behavioral change.

Reply:
Makes sense. I have separated the checks to see at one glance if message or type is wrong